<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DisplayWindow</class>
    <widget class="QMainWindow" name="DisplayWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1280</width>
                <height>800</height>
            </rect>
        </property>
        <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
        </property>
        <property name="windowTitle">
            <string>Affichage</string>
        </property>
        <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
        </property>
        <property name="documentMode">
            <bool>true</bool>
        </property>
        <property name="tabShape">
            <enum>QTabWidget::Triangular</enum>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <widget class="QStackedWidget" name="PageManager">
                        <widget class="QWidget" name="pageEventTitle">
                            <layout class="QGridLayout" name="gridLayout_2">
                                <item row="0" column="0" rowspan="2">
                                    <widget class="QLabel" name="ET_OrganizerLogo">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>1</horstretch>
                                                <verstretch>1</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>200</width>
                                                <height>200</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>TextLabel</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1" colspan="3">
                                    <spacer name="horizontalSpacer">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>695</width>
                                                <height>100</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="0" column="4" colspan="2">
                                    <widget class="QLabel" name="ET_OrganizerName">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>5</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>20</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Le Sou des écoles Genay</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1" colspan="4">
                                    <widget class="QLabel" name="ET_EventTitle">
                                        <property name="font">
                                            <font>
                                                <pointsize>35</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Le Loto du sou des écoles</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="5" rowspan="2">
                                    <spacer name="horizontalSpacer_2">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>161</width>
                                                <height>100</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="2" column="1" rowspan="2" colspan="4">
                                    <widget class="QLabel" name="ET_EventLogo">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>5</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>200</width>
                                                <height>200</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>TextLabel</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="3" column="0">
                                    <spacer name="horizontalSpacer_5">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>197</width>
                                                <height>100</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="3" column="5">
                                    <spacer name="horizontalSpacer_4">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>161</width>
                                                <height>100</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="4" column="0" colspan="2">
                                    <widget class="QLabel" name="ET_EventLocation">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>3</horstretch>
                                                <verstretch>2</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>20</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Espace Vicard</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="4" column="2">
                                    <spacer name="horizontalSpacer_3">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>486</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="4" column="3" colspan="3">
                                    <widget class="QLabel" name="ET_EventDate">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>3</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>20</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>28 Novembre 2021</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                        <widget class="QWidget" name="pageRoundTitle">
                            <widget class="QLabel" name="label_7">
                                <property name="geometry">
                                    <rect>
                                        <x>300</x>
                                        <y>250</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Round Title</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                        <widget class="QWidget" name="pageRoundRunning">
                            <widget class="QLabel" name="label_8">
                                <property name="geometry">
                                    <rect>
                                        <x>310</x>
                                        <y>230</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Round Running</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                        <widget class="QWidget" name="pageRoundPause">
                            <widget class="QLabel" name="label_9">
                                <property name="geometry">
                                    <rect>
                                        <x>340</x>
                                        <y>250</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Round Pause</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                        <widget class="QWidget" name="pageRoundEnd">
                            <widget class="QLabel" name="label_10">
                                <property name="geometry">
                                    <rect>
                                        <x>310</x>
                                        <y>170</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Round End</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                        <widget class="QWidget" name="pageEventPause">
                            <widget class="QLabel" name="label_11">
                                <property name="geometry">
                                    <rect>
                                        <x>330</x>
                                        <y>180</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Event Pause</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                        <widget class="QWidget" name="pageEventEnd">
                            <widget class="QLabel" name="label_12">
                                <property name="geometry">
                                    <rect>
                                        <x>300</x>
                                        <y>200</y>
                                        <width>601</width>
                                        <height>221</height>
                                    </rect>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>45</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Event End</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>true</bool>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                                <property name="wordWrap">
                                    <bool>true</bool>
                                </property>
                                <property name="textInteractionFlags">
                                    <set>Qt::NoTextInteraction</set>
                                </property>
                            </widget>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
